packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -1110,6 +1114,8 @@ class VideoViewport extends Viewport implements IVideoViewport {
       element: this.element,
       viewportId: this.id,
       viewport: this,
+      imageIndex: this.getCurrentImageIdIndex(),
```

Comment:
Need these for updating the scroll bar.  The integer frame index needs to be used so that every frame is accessible.